README.md

```diff
@@ -367,22 +367,74 @@ Address Family Identifier (AFI). Required. Valid values are `ipv4` and `ipv6`.
 ##### `safi`
 Sub Address Family Identifier (SAFI). Required. Valid values are `unicast` and `multicast`.
 
+##### `additional_paths_install`
+install a backup path into the forwarding table and provide prefix
+'independent convergence (PIC) in case of a PE-CE link failure.
+Valid values are true, false, or 'default'
+
+##### `additional_paths_receive`
+Enables the receive capability of additional paths for all of the
+neighbors under this address family for which the capability has
+not been disabled.  Valid values are true, false, or 'default'
+
+##### `additional_paths_selection`
+Configures the capability of selecting additional paths for a prefix.
+Valid values are a string defining the name of the route-map.
+
+##### `additional_paths_send`
+Enables the send capability of additional paths for all of the
+neighbors under this address family for which the capability has
+not been disabled. Valid values are true, false, or 'default'
+
 ##### `client_to_client`
-`client-to-client reflection`. Valid values are true and false.
+Configure client-to-client route reflection. Valid values are true and false.
+
```

Comment:
Consider adding comments to these dampening props to indicate whether they are optional or exclusive with other properties; ie. routemap may not be used when the timer properties are specified. Or else maybe add an entry before the others like this:

##### Dampening Properties
Note: dampening_routemap is mutually exclusive with dampening_half_time, reuse_time, suppress_time and max_suppress_time.